client/signup/steps/p2-site/index.jsx

```diff
@@ -461,11 +461,13 @@ class P2Site extends Component {
 	render() {
 		return (
 			<P2StepWrapper
+				className="p2-site__create"
 				flowName={ this.props.flowName }
 				stepName={ this.props.stepName }
 				positionInFlow={ this.props.positionInFlow }
-				headerText={ this.props.translate(
-					'Share, discuss, review, and collaborate across time zones, without interruptions.'
+				headerText={ this.props.translate( 'Create a workspace' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Create a new workspace' )` *ES Score: 6*